coredocument/coredocument.proto

```diff
@@ -54,9 +54,8 @@ message CoreDocument {
   bytes data_root = 5 [
     (proofs.exclude_from_tree) = true
   ];
-  // Signatures
-  // Signatures of the signature_root by centrifuge identities. This array should be sorted by the Centrifuge ID.
-  repeated Signature signatures = 6 [
+  // signature_data of the signature_root by centrifuge identities.
+  SignatureData signature_data = 6 [
```

Comment:
Needs  to be excluded from tree otherwise it will be included in the coredoc tree.

```
[
     (proofs.exclude_from_tree) = true
   ];
```

Reply:
already excluded